.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
```

Comment:
- Consider using a more descriptive variable name instead of COMMIT_DETAILS to improve readability.